scripts/pi-hole/php/add.php

```diff
@@ -34,12 +34,10 @@
         }
         break;
     case "wild":
-        if(!isset($_POST["auditlog"]))
-            echo exec("sudo pihole -wild -q ${_POST['domain']}");
-        else
+        if(file_put_contents($regexfile, $_POST['domain']."\n", FILE_APPEND) === FALSE)
         {
-            echo exec("sudo pihole -wild -q -n ${_POST['domain']}");
```

Comment:
We might want to also reload the regex filters through the API after adding the line.

Reply:
This condition was never fulfilled. We don't (and never did) support adding a domain as a wildcard from the audit log page. There is even a filter on this page that ensures that we don't even display the button for wildcard blocked domains.